package.json

```diff
@@ -23,8 +23,8 @@
     "clean:lerna": "lerna clean --yes",
     "clean:files": "rimraf coverage node_modules results",
     "copy-common-files": "sh ./scripts/copy-common-files.sh",
-    "format": "yarn format:prettier --write",
-    "format:prettier": "prettier '**/*.{js,md,mdx,scss,ts}' '!**/{build,es,lib,storybook,ts,umd,coverage}/**' '!README.md'",
+    "format": "run-s 'lint:es --fix' format:prettier",
```

Comment:
Is the --write not needed in this command?

Reply:
Yes -- I moved it to the end of `format:prettier`, on the next line vvv.  There's no use ever using prettier without the --write, cos it just dumps stuff on the console when you do that.